config/transformer/models_list

```diff
@@ -9,3 +9,6 @@ openconfig-interfaces-annot.yang
 openconfig-if-ip.yang
 openconfig-if-aggregate.yang
 openconfig-interfaces-annot.yang
+openconfig-mclag.yang
+openconfig-mclag-annot.yang
+sonic-mclag.yang
```

Comment:
no need to add sonic yang file name here

Reply:
remove sonic yang file from here. 
@kwangsuk , @amrutasali please approve the pull request if there are no further comments